src/mame/shared/xbox_nv2a.cpp

```diff
@@ -6,7 +6,14 @@
 #include <bitset>
 #include <cfloat>
 
-//#define LOG_NV2A
+#define LOG_NV2A         (1U << 1)
+#define LOG_NV2A_VERBOSE (1U << 2)
+
+#define VERBOSE (0)
+#define LOG_OUTPUT_FUNC machine().logerror
+#include "logmacro.h"
```

Comment:
This isn’t going to have the desired effect any more.  The intention of this was so that if `-DLOG_NV2A` was added to `ARCHOPTS` or `CXXFLAGS`, logging would be enabled in this file.  To get the same effect, you’d need to do something like:
```c++
#ifdef LOG_NV2A
#define VERBOSE (LOG_GENERAL)
#else
#define VERBOSE (0)
#endif
```
and then use `LOG` to log messages.

Reply:
Solved as suggested. Keeping `LOG_NV2A_VERBOSE` for what were previously commented-out `printf`s.